src/main/java/io/fabric8/jenkins/openshiftsync/PipelineJobListener.java

```diff
@@ -119,9 +122,23 @@ public void onDeleted(Item item) {
 				BuildConfig buildConfig = getAuthenticatedOpenShiftClient().buildConfigs().inNamespace(namespace)
 						.withName(buildConfigName).get();
 				if (buildConfig != null) {
+				    boolean generatedBySyncPlugin = false;
+				    Map<String,String> annotations = buildConfig.getMetadata().getAnnotations();
+				    if (annotations != null) {
+				        generatedBySyncPlugin = Annotations.GENERATED_BY_JENKINS.equals(annotations.get(Annotations.GENERATED_BY));
+				    }
 					try {
```

Comment:
some more weird indenting

Reply:
its for format with 80 chars a line thing